src/changeset/mod.rs

```diff
@@ -1,7 +1,10 @@
-use crate::{kv::*, models::*, CursorDupSort, *};
-use arrayref::array_ref;
+use crate::{
+    kv::{tables::BitmapKey, *},
+    models::*,
+    CursorDupSort,
+};
 use async_trait::async_trait;
-use bytes::Bytes;
+use roaring::RoaringTreemap;
```

Comment:
what's the benefit of changing these two?

Reply:
RoaringTreemap is now part of table definition and parsed in database layer, not in changesets.